01_Introduction_to_Citsci_Pipeline.ipynb

```diff
@@ -147,26 +147,57 @@
     "\n",
     "Supply the email associated with your Zooniverse account and project slug below. \n",
     "\n",
-    "A \"slug\" is the string of your Zooniverse username and your project name without the leading forward slash, for instance: \"username/project-name\". [Click here for more details](https://www.zooniverse.org/talk/18/967061?comment=1898157&page=1).\n"
+    "A \"slug\" is the string of your Zooniverse username and your project name without the leading forward slash, for instance: \"username/project-name\". [Click here for more details](https://www.zooniverse.org/talk/18/967061?comment=1898157&page=1).\n",
+    "\n",
+    "**If you have __not__ yet created a Zooniverse project do not provide a `slug_name` and run the subsequent cell**"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "0edb67a4-1508-4b90-872c-8c42e642126b",
+   "id": "a577e133-35ae-4e54-9ee2-727f3dce996c",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "email = \"beckynevin@gmail.com\"\n",
-    "slug_name = \"rebecca-dot-nevin/test-project\"\n",
+    "email = \"\"\n",
+    "slug_name = \"\"\n",
     "print(\"Loading and running utilities to establish a link with Zooniverse\")\n",
     "print(\"Enter your Zooniverse username followed by password below\")\n",
     "cit_sci_pipeline = pipeline.CitSciPipeline()\n",
     "cit_sci_pipeline.login_to_zooniverse(slug_name, email)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "081794cf-f493-4832-9c1c-6641bcd9b4ed",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2024-06-10T18:31:30.334716Z",
+     "iopub.status.busy": "2024-06-10T18:31:30.333633Z",
+     "iopub.status.idle": "2024-06-10T18:31:30.682149Z",
+     "shell.execute_reply": "2024-06-10T18:31:30.681257Z",
+     "shell.execute_reply.started": "2024-06-10T18:31:30.334679Z"
+    },
+    "tags": []
+   },
+   "source": [
+    "**Run the following cell if you would like to create a new Zooniverse project**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "23a9b8b6-52b9-4211-81a0-e6ceee9fd08f",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "cit_sci_pipeline.project = cit_sci_pipeline.create_new_project_from_template()"
```

Comment:
Something else to discuss is that people might still run this cell even if they do define a slugname. In that case, we would probably need to add some other controls, because do we want people to still be able to create a template project even if they separately defined the slug?

Reply:
We'll address this in a future PR about guardrails.